src/components/wallet/methods/updateAccount.ts

```diff
@@ -28,11 +15,21 @@ export default async function updateAccount(this: Wallet) {
       .accounts()
       .accountId(this.account.publicKey)
       .call()
+    let claimableBalanceResp = await this.server
+      .claimableBalances()
+      .claimant(this.account.publicKey)
+      .call()
+    // claimableBalanceResp.records.forEach((record) => {
```

Comment:
Hmm so I think claimable balance should still be logged so the user knows they have claimable balances outstanding. If they aren't logged, there is no visual indication that they exist right? I guess ultimately we'll have a button for claiming them, which should have a visual indicator that the claimable balance is available.

Reply:
Yes I'm happy to display the balances in the logger as well.
I thought at first it was a bit redundant since you can see the same results with the `Show Claimable Balance Details` button.

> I guess ultimately we'll have a button for claiming them, which should have a visual indicator that the claimable balance is available.

I do think that there is a path where we dont have this  `Show Claimable Balance Details` button and instead just have `"Claim Outstanding Balances"` that will show and allow the user to claim them.